prep-javascript-variables/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+   var fullName = 'Jessica Meyer';
```

Comment:
make sure to declare all of these variables using the `const` keyword except for never which must be declared with `let`